polls/models.py

```diff
@@ -0,0 +1,68 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from wagtail.wagtailcore.models import Page
+from wagtail.wagtailadmin.edit_handlers import FieldPanel
+from wagtail.wagtailsearch import index
+
+from molo.core.models import LanguagePage
+
+
+class LanguagePage(LanguagePage):
+    LanguagePage.subpage_types = [
+        'core.HomePage',
+        'core.SectionPage',
+        'core.FooterPage',
+        'polls.Question'
+    ]
+
+
+class Question(Page):
```

Comment:
@smn I see the _why_. This whole `Page` model that Wagtail enforces is one of the reasons I'm unhappy with Wagtail. Until we've done more research into ways of extending it, this approach will have to do.